File: app/Models/Remote.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Concerns\HasUlids;
use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Remote extends Model
{
    /** @use HasFactory<\Database\Factories\RemoteFactory> */
    use HasUlids, HasFactory;

    /**
     * The attributes that are mass assignable.
     *
     * @var list<string>
     */
    protected $fillable = [
        'id',
        'name',
        'provider',
        'repository',
        'branch',
    ];
}

File: pkg/store/remote.go
package store

import "time"

type Remote struct {
	ID         string    `json:"id" db:"id"`
	Name       string    `json:"name" db:"name"`
	Repository string    `json:"repository" db:"repository"`
	Branch     string    `json:"branch" db:"branch"`
	Provider   string    `json:"provider" db:"provider"`
	CreatedAt  time.Time `json:"created_at" db:"created_at"`
	UpdatedAt  time.Time `json:"updated_at" db:"updated_at"`
}
